ngi_reports/stockholm/project_summary.py

```diff
@@ -208,11 +219,11 @@ def __init__(self, config, LOG, working_dir, **kwargs):
                 try:
                     if re.sub('_+','.',stat['Project'],1) != self.project_name and stat['Project'] != self.project_name:
                         continue
-                    sample, lane = (stat['Sample'] if seq_plat == "HiSeqX" else stat['Sample ID'], stat['Lane'])
+                    sample, lane = (stat['Sample'] if fc['db'] == "x_flowcell" else stat['Sample ID'], stat['Lane'])
                     try:
-                        if seq_plat in ["HiSeq2500", "MiSeq"]:
+                        if fc['db'] == "flowcell":
```

Comment:
@vezzi : Yes, I think for two reasons. 
* (Mainly) to be able to generate reports for old projects as well if needed 
* also as a precaution in case if any project have fell in gap during the switch to `TACA` (though it is unlikely as we were update both databases for past sometime).